Clarify `suppress` description in Voice State (#5216)
hammerandchisel/discord-api-docs: docs/resources/Voice.md

| session_id                 | string                                                           | the session id for this voice state                 |
| deaf                       | boolean                                                          | whether this user is deafened by the server         |
| mute                       | boolean                                                          | whether this user is muted by the server            |
| self_deaf                  | boolean                                                          | whether this user is locally deafened               |
| self_mute                  | boolean                                                          | whether this user is locally muted                  |
| self_stream?               | boolean                                                          | whether this user is streaming using "Go Live"      |
| self_video                 | boolean                                                          | whether this user's camera is enabled               |

> | suppress                   | boolean                                                          | whether this user does not have permission to speak |

Reviewer: ```suggestion
| suppress                   | boolean                                                          | whether this user is denied permission to speak |
```

i think your edit is definitely an improvement—suggesting a small edit to align it with other boolean language ("is X")—i think including "does not have" will cause a tiny bit of confusion. feel free to edit further, im not tied to my suggestion.

also sorry this is gunna mess up your table formatting 😓

Author: Sure thing, English is not my native though.

Which one is better? `whether this user is denied to speak` or `whether this user permission is denied to speak`

Reviewer 2: something like ''whether this user's permission to speak is denied" sounds better
